GymApp/GymAppApi/Program.cs

```diff
@@ -23,6 +25,8 @@
     app.UseSwaggerUI();
 }
 
+app.UseCors("DefaultPolicy");
```

Comment:
just recommendation: 
reuse it using variable (const, for example). because you share it in two different files and if suddenly want to change it in one file, you can forget to do it in another one. as a result, you will get bug. so, it's better to avoid this situation